gen/definitions/high_availability.yaml

```diff
@@ -0,0 +1,145 @@
+---
+name: Device HA Pairs
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/devicehapairs/ftddevicehapairs
+data_source_name_query: true
+doc_category: Device
+attributes:
+  - model_name: name
+    type: String
+    mandatory: true
+    description: The name of the access control policy.
+    example: FTD_HA
+  - model_name: type
+    type: String
+    value: DeviceHAPair
+  - model_name: id
+    type: String
+    mandatory: true
+    tf_name: primary_device_id
+    data_path: [primary]
+    description: ID of primary FTD in the HA Pair.
+    example: <FTD1_ID>
```

Comment:
Can you put here some fake UUID string? This makes the example sound more real.

Reply:
done